client/components/note.jsx

```diff
@@ -160,6 +183,9 @@ class Note extends React.Component {
   }
 
   render() {
+    if (this.state === null) {
+      return null;
+    }
```

Comment:
you don't need to check this, this is fulfilled on line 277/303

Reply:
you are exactly right.  i'll remove these lines from the pull request.